gammapy/conftest.py

```diff
@@ -12,6 +12,10 @@
 ## from the list of packages for which version numbers are displayed
 ## when running the tests ... this was added in Astropy 1.0
 try:
+    # Removing h5py for now to get rid of this travis-ci fail
+    # https://travis-ci.org/gammapy/gammapy/jobs/86565582#L1074
+    del PYTEST_HEADER_MODULES['h5py']
```

Comment:
@cdeil - I think it's not worth trying to get a workaround. I heard back from Continuum, and the h5py package will be fixed today.